app/helpers/check_answers_helper.rb

```diff
@@ -56,11 +56,12 @@ def condition_not_met(case_log, question_key, question, condition)
       operator = condition[/[<>=]+/].to_sym
       operand = condition[/\d+/].to_i
       case_log[question_key].blank? || !case_log[question_key].send(operator, operand)
+    when "text"
+      case_log[question_key].blank? || !condition.include?(case_log[question_key])
     when "radio"
       case_log[question_key].blank? || !condition.include?(case_log[question_key])
     else
-      raise "Not implemented yet"
-    end
+      raise "Not implemented yet" end
```

Comment:
Don't think you need the `end` here do you?

Reply:
Do need it but moved it onto correct line